web/src/components/molecules/Common/WorkspaceMenu/index.tsx

```diff
@@ -48,6 +48,12 @@ const WorkspaceMenu: React.FC<Props> = ({
       icon: <Icon icon="api" />,
       show: "both" as MenuShowType,
     },
+    {
+      label: t("My Integration"),
+      key: "my-integration",
+      icon: <Icon icon="api" />,
+      show: "both" as MenuShowType,
```

Comment:
Can you remove the `as XxxxType` on all of these items. Since you are typing the items array it shouldn't be necessary

Reply:
@KaWaite , it is necessary here, but I will consider solving this in my dashboard refactoring PR, in which we discussed menus.

Reply:
Can you please explain WHY it is necessary, because for me I feel like since the array is typed (`WorkspaceItemType[]`), the `show` field should be okay as long as the value is `"both"` or `"personal"` or `"notPersonal"`.

Reply:
@KaWaite , it conflicts with `antd` menu type and asks for additional properties for some reason, so linter doesn't work as expected.
But I will address this when changing the menu code, and think of a solution.